libpod/storage.go

```diff
@@ -59,7 +62,7 @@ func (metadata *RuntimeContainerMetadata) SetMountLabel(mountLabel string) {
 
 // CreateContainerStorage creates the storage end of things.  We already have the container spec created
 // TO-DO We should be passing in an Image object in the future.
-func (r *storageService) CreateContainerStorage(ctx context.Context, systemContext *types.SystemContext, imageName, imageID, containerName, containerID, mountLabel string, options *storage.ContainerOptions) (cinfo ContainerInfo, err error) {
+func (r *storageService) CreateContainerStorage(ctx context.Context, systemContext *types.SystemContext, imageName, imageID, containerName, containerID, mountLabel string, options storage.ContainerOptions) (cinfo ContainerInfo, err error) {
```

Comment:
Do we still want mountLabel in the arguments here, since it's being assigned by c/storage now?

Reply:
Good Catch. This caused me to find another issue with Inspect.  Fixed both.